data-protocols/ids/ids-core/src/test/java/org/eclipse/dataspaceconnector/ids/core/message/DataRequestMessageSenderTest.java

```diff
@@ -0,0 +1,123 @@
+ *
+ */
+package org.eclipse.dataspaceconnector.ids.core.message;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.javafaker.Faker;
+import de.fraunhofer.iais.eis.ArtifactRequestMessageImpl;
+import okhttp3.Call;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okio.Buffer;
+import org.easymock.Capture;
+import org.easymock.EasyMock;
+import org.eclipse.dataspaceconnector.spi.iam.IdentityService;
+import org.eclipse.dataspaceconnector.spi.iam.TokenResult;
+import org.eclipse.dataspaceconnector.spi.monitor.Monitor;
+import org.eclipse.dataspaceconnector.spi.security.Vault;
+import org.eclipse.dataspaceconnector.spi.transfer.store.TransferProcessStore;
+import org.eclipse.dataspaceconnector.spi.types.domain.transfer.DataAddress;
+import org.eclipse.dataspaceconnector.spi.types.domain.transfer.DataRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+import static java.util.Map.entry;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.easymock.EasyMock.capture;
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.newCapture;
+import static org.easymock.EasyMock.niceMock;
+import static org.easymock.EasyMock.replay;
+import static org.easymock.EasyMock.verify;
+import static org.eclipse.dataspaceconnector.ids.spi.Protocols.IDS_REST;
+
+class DataRequestMessageSenderTest {
+
+    public static final String DESTINATION_KEY = "dataspaceconnector-data-destination";
+    public static final String PROPERTIES_KEY = "dataspaceconnector-properties";
+    static Faker faker = new Faker();
+    private final String connectorAddress = "http://" + faker.internet().url();
+    private final String connectorId = faker.internet().url();
+    private final String processId = faker.internet().uuid();
+    private final ObjectMapper mapper = new ObjectMapper();
+
+
+    private DataRequestMessageSender sender;
+    private OkHttpClient httpClient;
+
+    @BeforeEach
+    public void setUp() {
+        httpClient = niceMock(OkHttpClient.class);
+
+        IdentityService identityService = niceMock(IdentityService.class);
+        TokenResult tokenResult = niceMock(TokenResult.class);
+        expect(tokenResult.getToken()).andReturn(faker.lorem().characters());
+        expect(identityService.obtainClientCredentials(connectorId)).andReturn(tokenResult);
+        replay(identityService, tokenResult);
+        sender = new DataRequestMessageSender(connectorId, identityService, niceMock(TransferProcessStore.class), EasyMock.mock(Vault.class), httpClient, mapper, niceMock(Monitor.class));
+    }
+
+    @Test
+    public void initiateIdsMessage() throws IOException {
+        DataRequest dataRequest = createDataRequest();
+        DataAddress dataDestination = dataRequest.getDataDestination();
+
+        // record
```

Comment:
I put these comments because it seemed to me that it’s a common approach for tests in ids module. And that they can help to separate the sections in the test, which improves the readability.
E.g. CatalogServiceImplTest, DescriptionHandlerTest, AssetToIdsRepresentationTransformerTest and other tests especially in ids-transform-v1 

Is there an agreement across the EDC repo, in which tests is it advisable to have the comments or it’s a matter of personal preference?

Reply:
I think they came from a single test and then some copy-and-paste work. I think they are pretty obvious so they can be deleted.